include/openPMD/RecordComponent.hpp

```diff
@@ -137,9 +156,38 @@ class RecordComponent : public BaseRecordComponent
         Extent,
         double targetUnitSI = std::numeric_limits< double >::quiet_NaN() );
 
+    /** Store a chunk of data
+     *
+     * shared_ptr< T > for data must be contiguous and large enough for extent
```

Comment:
> and large enough for extent

This does not apply here or is misworded. It must have the same number of entries as the cumulative product of extent. *large enough* does not portray this fact.

Reply:
ok, makes sense to be a more explicit.